hnn_core/network.py

```diff
@@ -219,47 +293,443 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
-    def _add_external_feeds(self):
-        """Legacy function, for backward compatibility with original HNN-GUI
+    def add_evoked_drive(self, name, *,
+                         mu=None, sigma=None, numspikes=None,
+                         weights_ampa=None, weights_nmda=None, location=None,
+                         space_constant=3., dispersion_time=0.1, seedcore=42):
+        """Add an 'evoked' external drive to the network
 
-        This function is called exactly once during initialisation
+        Parameters
+        ----------
+        name : str
+            Unique name for drive
+        mu : float
+            Mean of Gaussian event time distribution (across target cells)
+        sigma : float
+            Standard deviation of event time distribution (across target cells)
+            Set to zero for "synchronous" evoked drive.
+        numspikes : int
+            Number of spikes at each target cell
+        weights_ampa : dict or None
+            Synaptic weights of AMPA receptors on each targeted cell type
+            (keys). Weights can be zero or left omitted.
+        weights_nmda : dict or None
+            Synaptic weights of NMDA receptors on each targeted cell type
+            (keys). Weights can be zero or left omitted.
```

Comment:
these weights also decay from origin by the way. See [here](https://github.com/jonescompneurolab/hnn-core/blob/master/hnn_core/cell.py#L351) and [here](https://jonescompneurolab.github.io/hnn-under_the_hood/03_architecture-connectivity/03_architecture-connectivity)

Reply:
Should we just rename `dispersion_time` (weird) to `synaptic_delays`, as I proposed originally? Then let `space_constant` do its thing on both (and add mention of it to docs).